core/trino-main/src/main/java/io/trino/memory/NodeMemoryConfig.java

```diff
@@ -42,6 +42,8 @@
 
     private DataSize heapHeadroom = DataSize.ofBytes(Math.round(AVAILABLE_HEAP_MEMORY * 0.3));
 
+    private DataSize maxTaskDescriptorStorageMemory = DataSize.ofBytes(Math.round(AVAILABLE_HEAP_MEMORY * 0.3));
```

Comment:
feels a lot

Reply:
Discussed offline. Reduced to `0.15` for now